src/components/Badge.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { HTMLAttributes, forwardRef } from 'react';
+import { FC } from 'react';
+import { VariantProps, cva } from 'class-variance-authority';
+import { twMerge } from 'tailwind-merge';
+
+const BadgeVariants = cva('w-fit px-2', {
+  variants: {
+    size: {
+      xs: 'text-xs leading-none',
+      sm: 'text-sm leading-[1.15]',
+      base: 'text-base leading-[1.15]',
+      lg: 'text-lg leading-[1.33]'
+    },
+    bcolor:{
+        '1':'bg-surface-1',
+        '2':'bg-surface-2',
+        '3':'bg-surface-3'
+    },
```

Comment:
I think we can just pass `className` with bg color as external prop. No need to limit developers here